lib/mpool/include/mpool/mpool.h

```diff
@@ -644,6 +644,16 @@ mpool_file_mmap(struct mpool_file *file, bool read_only, int advice, char **addr
 size_t
 mpool_file_size(struct mpool_file *file);
 
+/**
+ * mpool_mcpath_is_fsdax() - is the mclass path on a DAX filesystem
+ *
+ * @dpath: media class path
+ * @isdax: return true if dax (output)
+ */
+merr_t
+mpool_mcpath_is_fsdax(const char *path, bool *isdax);
```

Comment:
daxfs instead of fsdax?

Reply:
"fsdax" is one of the modes in which a pmem can be configured. Will leave it as is.